proposals/lambda-method-group-defaults.md

```diff
@@ -69,41 +81,54 @@ app.MapPost("/todos/{id}", todoHandler);
 ```
 
 ## Current Behavior
-Currently, when a user implements a lambda with a default value, the compiler raises an error stating that default values are not allowed. 
+Currently, when a user implements a lambda with a default or `params` parameter, the compiler raises an error stating that it's not allowed. 
```

Comment:
>default or `params` parameter [](http://example.com/codeflow?start=50&length=29)

nit and not directly related to this PR: the spec uses "default_argument" and "optional parameter", but I've not seen "default parameter" before.

Reply:
Good catch, I have also been wondering whether to change that. So, now I did: 9c3e329cb5c26b955774f73f2f89714c2a28dbbb